WordPress/Classes/Services/BlogService.m

```diff
@@ -20,14 +20,28 @@
 #import "NSString+XMLExtensions.h"
 #import "TodayExtensionService.h"
 
+NSString *const LastUsedBlogURLDefaultsKey = @"LastUsedBlogURLDefaultsKey";
+NSString *const BlogEntity = @"Blog";
+NSString *const EditPostViewControllerLastUsedBlogURLOldKey = @"EditPostViewControllerLastUsedBlogURL";
+NSString *const BlogName = @"blogName";
+NSString *const GetFeatures = @"wpcom.getFeatures";
+NSString *const VideopressEnabled = @"videopress_enabled";
+NSString *const XmlRpc = @"xmlrpc";
+NSString *const Standard = @"standard";
+NSString *const MinimumVersion = @"3.6";
+NSString *const DateCreatedGmt = @"date_created_gmt";
+NSString *const TimeZoneName = @"timezone";
```

Comment:
Sure, I'm totally glad to take advice on the naming. If I were to move these to external constants in Blog.m, should I also replace their usages throughout the project or does that fall into the 'too-broad' category in terms of cluttering the pull request?